src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +45,10 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("FROM Movie", Movie.class).list();
+        } catch (Exception e) {
+            throw new RuntimeException("Error occurred while trying to get all movies", e);
```

Comment:
The exception thrown here should be consistent with the one used in other DAO methods. Consider using `DataProcessingException` for uniformity and more accurate exception handling.